keras/src/ops/numpy_test.py

```diff
@@ -9507,3 +9511,24 @@ def call(self, x):
         model.compile(jit_compile=jit_compile)
 
         model.predict(np.random.randn(1, 8))
+
+    def test_tile_shape_inference_in_layer(self):
+        """Test that ops.tile properly infers output shape in a Layer.
+
+        This is a regression test for issue #20914 where TensorFlow backend
+        would return all-None shapes when tile was called inside a Layer's
+        call method with concrete integer repeats.
+        """
+
```

Comment:
Remove this comment. While it's relevant in the context of the PR, if we did this for every PR, it would really add up and make the code less readable.

We can always recover the context of this test by using blame / history.